REVIEWING.md

```diff
@@ -0,0 +1,62 @@
+# Reviewing
+
+In the metadata repository, we accept contributions from 3rd parties. Per Oracle policies, we need to do a proper review of such code,
+including licensing and security checks. This document should serve as a guideline for reviewers to simplify and harmonize those reviews.
+
+## Checklist
+The first step of every review is to verify the checklist from the pull request description. Once the contributor wants to open 
+a pull request [this](pull_request_template.md) checklist will be automatically added to the pull request description.
```

Comment:
SEO tip: don't use "this" as link target, but rephase so that "checklist" or "this checklist" is the target. More obvious to the reader, more obvious to Google :-)